packages/office-ui-fabric-react/src/components/Layer/examples/Layer.Hosted.Example.tsx

```diff
@@ -1,18 +1,49 @@
 import * as React from 'react';
 import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
 import { Layer, LayerHost } from 'office-ui-fabric-react/lib/Layer';
-import { AnimationClassNames, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { AnimationClassNames, mergeStyleSets, getTheme, IStyleSet } from 'office-ui-fabric-react/lib/Styling';
 import { getId, css } from 'office-ui-fabric-react/lib/Utilities';
 import { IToggleStyles } from 'office-ui-fabric-react/lib/Toggle';
-import { IStyleSet } from 'office-ui-fabric-react/lib/Styling';
-import * as styles from './Layer.Example.scss';
 
 const toggleStyles: Partial<IStyleSet<IToggleStyles>> = {
   root: { margin: '10px 0' }
 };
 
-const rootClass = mergeStyles({
-  selectors: { p: { marginTop: 30 } }
+const theme = getTheme();
+const classNames = mergeStyleSets({
```

Comment:
do we need this amount of styling in the example? it's fine to leave it to keep it looking nice,  but just a fleeting thought.

Reply:
I agree the amount of styling (especially for identifying the layer host) is unfortunate, but I think in this case it's best to leave it since I can't think of an equally clear way to identify the LayerHost which requires less styling (a label would be much less clear).
http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/10143/merge/fabric-website-resources/dist/index.html#/examples/layer

![image](https://user-images.githubusercontent.com/5864305/63076613-a7e5f100-beea-11e9-9505-dbebfc54e9e1.png)